src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -232,3 +239,53 @@ def _query_organizations(
             return [r.organization for r in results if scope in r.get_scopes()]
 
         return [r.organization for r in results]
+
+    def get_teams(self, team_ids: Iterable[int]) -> List[ApiTeam]:
+        teams: Iterable[Team] = Team.objects.filter(id__in=list(team_ids))
+        return [self._serialize_team(team) for team in teams]
+
+    @staticmethod
+    def _deserialize_member_flags(flags: ApiOrganizationMemberFlags) -> int:
+        return flags_to_bits(flags.sso__linked, flags.sso__invalid, flags.member_limit__restricted)
+
+    def add_organization_member(
+        self,
+        *,
+        organization: ApiOrganization,
+        user: APIUser,
+        flags: ApiOrganizationMemberFlags | None,
+        role: str | None,
+    ) -> ApiOrganizationMember:
+        member = OrganizationMember.objects.create(
+            organization_id=organization.id,
+            user_id=user.id,
+            flags=self._deserialize_member_flags(flags) if flags else 0,
+            role=role or organization.default_role,
+        )
+        return self._serialize_member(member)
+
+    def add_team_member(
+        self, *, team: ApiTeam, organization_member: ApiOrganizationMember
+    ) -> ApiTeamMember:
+        omt = OrganizationMemberTeam.objects.create(
+            team_id=team.id, organizationmember_id=organization_member.id
+        )
+        project_ids = ()  # TODO?
+        return self._serialize_team_member(omt, project_ids)
+
+    def get_audit_log_data(
+        self, *, organization_member: ApiOrganizationMember
+    ) -> Mapping[str, Any]:
+        from sentry.services.hybrid_cloud.user import user_service
+
+        user = user_service.get_user(user_id=organization_member.user_id)
+        teams = self.get_teams(mt.team_id for mt in organization_member.member_teams)
```

Comment:
To do: See if this is redundant to an existing service. Note that we already have an org ID.